scene/3d/audio_stream_player_3d.cpp

```diff
@@ -33,6 +33,9 @@
 #include "scene/3d/area.h"
 #include "scene/3d/camera.h"
 #include "scene/main/viewport.h"
+
+#define ATT_DIST_EPSILON 0.00001
```

Comment:
Can't you use the existing `CMP_EPSILON` directly?

Reply:
Where is `CMP_EPSILON` located? Or is it in the global scope so it can be written withoud field access in another place?

Reply:
It's in `core/math/math_defs.h`, but that's included virtually everywhere via `math_funcs.h` (which is included in `core/ustring.h` and `core/variant.h` for example.
If it doesn't compile as is, you can add `#include "core/math/math_defs.h"` of course, but I think it likely won't be necessary.

Reply:
Done.